generator/src/test/java/com/scottlogic/deg/generator/cucumber/Null.feature

```diff
@@ -10,12 +10,9 @@ Feature: User can specify that a field is null or absent
       | foo  |
       | null |
 
-
-  @ignore @bug
   Scenario: Negating the 'null' operator generates non-null values
     Given foo is in set:
       | "notNull" |
-      | null      |
```

Comment:
This means an empty set, is this correct?

Reply:
As discussed this test has been rewritten to  check that null is not produced when null is negated and a new test has been added to ensure that no data is created when a field must be null and must not be null that no data is generated.